Improve FIP-0014 (#85)
filecoin-project/FIPs: FIPS/fip-0014.md

### Time for development of custom hardware
It is estimated that manufacturing a chip from scratch could take [1 to 2 years](https://www.quora.com/How-long-does-it-take-to-build-a-computer-chip-from-scratch). By this point, most, if not all, v1 sectors will have expired.

### Securing production of custom hardware
There are mainly two foundries someone could get their chip design produced, namely from TSMC or Samsung. Attackers will be competing with all smart devices manufacturers for "production cap". [Months of waiting](https://www.dvhardware.net/article72264.html) time is the norm given how the market is craving smart devices nowadays.

### Incentives of attackers
Under the circumstances mentioned above, there wouldn't be any incentives for attackers to invest huge amount of time and money just to fake a discontinued proof.

### Core dev meeting
Many of the security considerations were also extensively discussed in [Filecoin Core Dev Meeting #13](https://github.com/filecoin-project/tpm/blob/master/Core%20Dev%20Meetings/Meeting%200013.md). Some of the key takeaways are...

> > We need to agree on final answers to:

Reviewer: ```suggestion
> Core devs need to agree on final answers to:
```

> - whether adopting this FIP is acceptable from a security perspective
> - whether the community's overall preference is in fact for this FIP
> 

> > Answers to both questions are tending towards "yes"

Reviewer: ```suggestion
> The answer for the first question is tending towards "yes", and the answer to the second question should be found out via a community poll
```

### Summary of security considerations
The general consensus is that it is not economically viable that an attacker could develop custom hardware using tech that is a year+ from now. Some highlights of community comments are by @neogeweb3 at [here](https://github.com/filecoin-project/FIPs/pull/75#discussion_r586057806) and by @steven004 at [here](https://github.com/filecoin-project/FIPs/issues/56#issuecomment-785715308).

## Incentive Considerations

> The majority of the people who sealed sectors using V1 proof for 180 days will now most likely to extend them to 540 days. 

Reviewer: ```suggestion
The majority of the people who sealed sectors using V1 proof for 180 days will now most likely to extend them up to 540 days. 
```

We expect such behaviour thanks to one of crucial points that @steven004 brought up, which is [fairness](https://github.com/filecoin-project/FIPs/issues/56#issuecomment-794778752). As summed up by Aayush in this [thread](https://filecoinproject.slack.com/archives/C01EU76LPCJ/p1615524308014600?thread_ts=1615283805.008800&cid=C01EU76LPCJ)...

> My personal opinion is that it’s a fairness question — folks who sealed 6 month v1 sectors thinking they could be easily extended later are now in the unfortunate position of not being able to do so (while those who sealed them for 18 months will continue to have power for them).

## Product Considerations
This FIP protects interests of miners who have contributed to Filecoin's growth since mainnet launch. 

> This FIP is both a both security/implementation decision and governance decision. The decision made in [v1.2.0](https://github.com/filecoin-project/lotus/releases/tag/v1.2.0) of not extending V1 proof sectors was not an obvious catch. It wasn't until @deltazxm tried to extend a V1 proof sector and got an [error message](https://github.com/filecoin-project/specs-actors/issues/1309) that people realized the decision had been made. What if we find a bug in V1.1 tomorrow? Do we silently let V1.1 proof expire too? We need to establish some kind of process to reach consensus on making decisions like this together with different kinds of stakeholders in the network especially when changing policy, network parameters or other equivalents.

Reviewer: ```suggestion
This FIP is both a both security/implementation decision and governance decision. The decision made in [v1.2.0](https://github.com/filecoin-project/lotus/releases/tag/v1.2.0) of not allowing extending V1 proof sectors was not an obvious catch. It wasn't until @deltazxm tried to extend a V1 proof sector and got an [error message](https://github.com/filecoin-project/specs-actors/issues/1309) that people realized the decision had been made. What if we find a bug in V1.1 tomorrow? Do we silently let V1.1 proof expire too? We need to establish some kind of process to reach consensus on making decisions like this together with different kinds of stakeholders in the network especially when changing policy, network parameters or other equivalents.
```

Author: good point!
